### DISCUSSION TITLE
Dunst break

### DISCUSSION THREAD

**Discussion by mahaveergurjar** [prasanthrangan/hyprdots#1348] [Q&A]
How to fix the dunst not show some icons 
![240420_18h43m49s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/57872034/933dc576-7b01-4d39-811e-5ee818dda252)

When i click on notifictions the notification rofi not open
How to open rofi notification 
@Skerse 
![240420_18h28m39s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/57872034/eef3c95f-8389-4db3-b792-bcc76442180d)

**Comment by dieBakterie** [comment]
For me this doesn't even works. When i click an notification it doesn't even open the Rofi window

**Reply by mahaveergurjar** [reply]
Yeah same for me so i asked
And also after update dunst 
brightness icon not show

**Reply by dieBakterie** [reply]
For me even the Window there in the Pic isn't showing up. When i click on a notification nothing Happens.
